api/target/kusttarget.go

```diff
@@ -337,6 +347,109 @@ func (kt *KustTarget) accumulateResources(
 	return nil
 }
 
+// isKustomizeBaseDirectory returns true if the directory pointed to be path contains
+// at least 1 recognized kustomization filename which may be used as a base.
+func (kt *KustTarget) isKustomizeBaseDirectory(path string) bool {
```

Comment:
There is a function `loadKustFile` in this file. Maybe you can leverage that. btw, `path` is already a directory when `kt.ldr.New` returns nil err.

Reply:
This is a sanity check for some of the tests which don't setup the data in a way that is recognized.

Reply:
please move this method to the fileloader.go

that should fix the FS exposure problem.

Reply:
Thanks.  Done.  Also moved Walk.  LMK if you think it makes sense.